sda-auth/main.go

```diff
@@ -390,6 +390,14 @@ func main() {
 	app.Get("/elixir/login", authHandler.getElixirLogin)
 	app.Get("/elixir/cors_login", authHandler.getElixirCORSLogin)
 
+	// Info endpoint
+	publicKey, err := readPublicKeyFile(authHandler.Config.C4ghPubKey)
```

Comment:
This should be cached so that we don't have to read the file every time someone calls the info endpoint.

Reply:
How is this called on every call? It's in main.